deployment/chainloop/templates/controlplane/config.secret.yaml

```diff
@@ -23,7 +23,7 @@ stringData:
   config.observability.yaml: |
     {{- include "chainloop.sentry" .Values.controlplane.sentry | nindent 4 }}
   {{- end }}
-  {{- if and .Values.controlplane.keylessSigning .Values.controlplane.keylessSigning.enabled }}
+  {{- if and .Values.controlplane.keylessSigning.enabled (eq "fileCA" .Values.controlplane.keylessSigning.backend) }}
```

Comment:
For next time. I think it's okay, and preferable in my case, to have this separated into two PRs. One for the feature and the other one for the deployment template.